checkpoint/runner_test.go

```diff
@@ -44,28 +66,59 @@ var allAccounts = []*types.Account{
 	{Layer: types.LayerID(7), Address: types.Address{4, 4}, NextNonce: 1, Balance: 31, TemplateAddress: &types.Address{3}, State: []byte("state47")},
 }
 
-func expectedCheckpoint(t *testing.T) *checkpoint.Checkpoint {
-	return &checkpoint.Checkpoint{
+func expectedCheckpoint(t *testing.T, snapshot types.LayerID, numAtxs int) *checkpoint.Checkpoint {
+	t.Helper()
+	result := &checkpoint.Checkpoint{
 		Version: "https://spacemesh.io/checkpoint.schema.json.1.0",
 		Data: checkpoint.InnerData{
 			CheckpointId: "snapshot-5",
-			Atxs: []checkpoint.ShortAtx{
-				toShortAtx(newvatx(t, allAtxs[3]), allAtxs[0].CommitmentATX, allAtxs[0].VRFNonce),
-				toShortAtx(newvatx(t, allAtxs[2]), allAtxs[0].CommitmentATX, allAtxs[0].VRFNonce),
-				toShortAtx(newvatx(t, allAtxs[4]), allAtxs[4].CommitmentATX, allAtxs[4].VRFNonce),
-				toShortAtx(newvatx(t, allAtxs[6]), allAtxs[5].CommitmentATX, allAtxs[5].VRFNonce),
-				toShortAtx(newvatx(t, allAtxs[5]), allAtxs[5].CommitmentATX, allAtxs[5].VRFNonce),
-			},
-			Accounts: []checkpoint.Account{
-				{types.Address{1, 1}.Bytes(), 111, 5, types.Address{2}.Bytes(), []byte("state15")},
-				{types.Address{2, 2}.Bytes(), 311, 14, types.Address{2}.Bytes(), []byte("state24")},
-				{types.Address{3, 3}.Bytes(), 124, 1, types.Address{3}.Bytes(), []byte("state35")},
-			},
 		},
 	}
+
+	if numAtxs < 2 {
+		require.Fail(t, "numEpochs must be at least 2")
+	}
+
+	atxs := make([]checkpoint.ShortAtx, 0, numAtxs*len(allAtxs))
+	for _, identity := range allAtxs {
```

Comment:
the map is identity to atxs.... the variable name should be atxs instead of identity?